examples/react/contexts/WalletSelectorExportContext.tsx

```diff
@@ -0,0 +1,158 @@
+import { setupSender } from "@near-wallet-selector/sender";
+import { setupMathWallet } from "@near-wallet-selector/math-wallet";
+import { setupNightly } from "@near-wallet-selector/nightly";
+import { setupMeteorWallet } from "@near-wallet-selector/meteor-wallet";
+import { setupWelldoneWallet } from "@near-wallet-selector/welldone-wallet";
+import { setupNightlyConnect } from "@near-wallet-selector/nightly-connect";
+import { setupNearFi } from "@near-wallet-selector/nearfi";
+import { setupWalletConnect } from "@near-wallet-selector/wallet-connect";
+import { setupCoin98Wallet } from "@near-wallet-selector/coin98-wallet";
+import { setupOptoWallet } from "@near-wallet-selector/opto-wallet";
+import { Loading } from "../components/Loading";
+
+declare global {
+  interface Window {
+    importSelector: WalletSelector;
+    ExportModal: WalletSelectorModal;
+  }
+}
+
+interface ExportAccountSelectorContextValue {
+  importSelector: WalletSelector;
+  ExportModal: WalletSelectorModal;
+  accounts: Array<AccountState>;
+  accountId: string | null;
+}
+
+const ExportAccountSelectorContext =
+  React.createContext<ExportAccountSelectorContextValue | null>(null);
+
+export const ExportAccountSelectorContextProvider: React.FC<{
+  children: ReactNode;
+}> = ({ children }) => {
+  const [importSelector, setSelector] = useState<WalletSelector | null>(null);
+  const [ExportModal, setModal] = useState<WalletSelectorModal | null>(null);
+  const [accounts, setAccounts] = useState<Array<AccountState>>([]);
+
+  const init = useCallback(async () => {
+    const _selector = await setupWalletSelector({
+      network: "testnet",
+      debug: true,
+      modules: [
+        ...(await setupDefaultWallets()),
+        setupNearWallet(),
+        setupSender(),
+        setupMathWallet(),
+        setupNightly(),
+        setupMeteorWallet(),
+        setupWelldoneWallet(),
+        setupHereWallet(),
+        setupCoin98Wallet(),
+        setupNearFi(),
+        setupOptoWallet(),
+        setupWalletConnect({
+          projectId: "c4f79cc...",
+          metadata: {
+            name: "NEAR Wallet Selector",
+            description: "Example dApp used by NEAR Wallet Selector",
+            url: "https://github.com/near/wallet-selector",
+            icons: ["https://avatars.githubusercontent.com/u/37784886"],
+          },
+        }),
+        setupNightlyConnect({
+          url: "wss://relay.nightly.app/app",
+          appMetadata: {
+            additionalInfo: "",
+            application: "NEAR Wallet Selector",
+            description: "Example dApp used by NEAR Wallet Selector",
+            icon: "https://near.org/wp-content/uploads/2020/09/cropped-favicon-192x192.png",
+          },
+        }),
+      ],
+    });
+    /**
+     * Insert list of accounts to be imported here
+     * accounts: [{ accountId: "test.testnet", privateKey: "ed25519:..."}, ...]
+     */
+    const _modal = setupExportSelectorModal(_selector, {
+      accounts: [],
```

Comment:
How will a user in a dApp set their accounts here?

It looks like only the dApp developer has control over this at the moment.

Reply:
dApp dev can use account-export part of code along with their own implementation of ability to dynamically pass `accountId` and `privateKey`. 

In terms of plan moving forward, `account-export` will be implemented on `near-wallet`. Part of the code from `near-wallet` will be dynamically create list of accounts (and their private keys stored in local storage) and feed them into account-export code. 

I would expect dApp devs to use account-export same way as above.